src/ItemGrid/index.ts

```diff
@@ -0,0 +1,4 @@
+import ItemGrid, { ItemGridProps } from './ItemGrid';
+
+export { ItemGridProps };
```

Comment:
We also need to export in the src/index (for consumer use) and add a link to it in the README.md

Reply:
Completely forgot about that part. Thanks for reminding.